packages/glibc_build237.rb

```diff
@@ -0,0 +1,323 @@
+            "
+          # install-symbolic-link segfaults on armv7l, but we're deleting
+          # the libraries anyways, so it doesn't matter.
+          system "sed -i 's,install-symbolic-link,/bin/true,g' ../Makefile"
+          system "sed -i 's,symbolic-link-prog := $(elf-objpfx)sln,symbolic-link-prog := /bin/true,g' ../Makerules"
+        when 'x86_64'
+          File.write('configparms', "slibdir=#{CREW_LIB_PREFIX}", mode: 'a+')
+          system "CFLAGS='-fuse-ld=mold -pipe -O2 -fipa-pta \
+          -fno-semantic-interposition -falign-functions=32 \
+          -fdevirtualize-at-ltrans' \
+            ../configure \
+            --prefix=#{CREW_PREFIX} \
+            --libdir=#{CREW_LIB_PREFIX} \
+            --with-headers=#{CREW_PREFIX}/include \
+            ac_cv_header_cpuid_h=yes \
+            ac_cv_lib_audit_audit_log_user_avc_message=no \
+            ac_cv_lib_cap_cap_init=no \
+            --disable-profile \
+            --disable-sanity-checks \
+            --disable-werror \
+            --enable-bind-now \
+            --enable-cet \
+            --enable-hacker-mode \
+            --enable-kernel=5.10 \
+            --enable-shared \
+            libc_cv_386_tls=yes \
+            libc_cv_arm_tls=yes \
+            libc_cv_asm_cfi_directives=yes \
+            libc_cv_broken_visibility_attribute=no \
+            libc_cv_c_cleanup=yes \
+            libc_cv_forced_unwind=yes \
+            libc_cv_gcc___thread=yes \
+            libc_cv_hashstyle=no \
+            libc_cv_mlong_double_128ibm=yes \
+            libc_cv_mlong_double_128=yes \
+            libc_cv_predef_fortify_source=no \
+            libc_cv_visibility_attribute=yes \
+            libc_cv_x86_64_tls=yes \
+            libc_cv_z_combreloc=yes \
+            libc_cv_z_execstack=yes \
+            libc_cv_z_initfirst=yes \
+            libc_cv_z_nodelete=yes \
+            libc_cv_z_nodlopen=yes \
+            libc_cv_z_relro=yes \
+            --with-binutils=binutils \
+            --with-bugurl=https://github.com/chromebrew/chromebrew/issues/new \
+            --without-cvs \
+            --without-gd \
+            --without-selinux \
+            "
+        end
+      end
+      system "make PARALLELMFLAGS='-j #{CREW_NPROC}' || make || make PARALLELMFLAGS='-j 1'"
+      if @libc_version.to_f >= 2.32
+        system "gcc -Os -g -static -o build-locale-archive ../fedora/build-locale-archive.c \
+          ../glibc_build/locale/locarchive.o \
+          ../glibc_build/locale/md5.o \
+          ../glibc_build/locale/record-status.o \
+          -I. -DDATADIR=\\\"#{CREW_PREFIX}/share\\\" -DPREFIX=\\\"#{CREW_PREFIX}\\\" \
+          -L../glibc_build \
+          -B../glibc_build/csu/ -lc -lc_nonshared"
+      end
+    end
+  end
+
+  def self.install
+    FileUtils.mkdir_p CREW_DEST_LIB_PREFIX
+    system "sed 's,/usr/#{ARCH_LIB}/libc_nonshared.a,#{CREW_LIB_PREFIX}/libc_nonshared.a,g' /usr/#{ARCH_LIB}/libc.so > #{CREW_DEST_LIB_PREFIX}/libc.so"
```

Comment:
Add to `def self.patch` section?

Reply:
And then a FileUtils.install here?

Reply:
Not sure.  Maybe you can patch the source prior to the build?

Reply:
Upon further reflection.... This is an intentional hack during the build process to let us work with the existing ChromeOS libraries... 

It's not really a patch, and it isn't really something where we can modify the source... We're just taking existing files and copying them over so we have them, with slight modifications so they refer to our paths...